ocaml/xapi/xapi_cluster.ml

```diff
@@ -63,9 +63,11 @@ let create ~__context ~network ~cluster_stack ~pool_auto_join ~token_timeout ~to
         Db.Cluster_host.create ~__context ~ref:cluster_host_ref ~uuid:cluster_host_uuid ~cluster:cluster_ref ~host ~enabled:true
           ~current_operations:[] ~allowed_operations:[] ~other_config:[];
         Xapi_cluster_host_helpers.update_allowed_operations ~__context ~self:cluster_host_ref;
-        D.debug "Created Cluster: %s and Cluster_host: %s" cluster_uuid cluster_host_uuid;
```

Comment:
What was wrong with the uuid?

Reply:
I think Jon said we should use opaquerefs in logging instead of uuids.